Gem/Code/Source/Components/UI/DemoPlacardComponent.cpp

```diff
@@ -30,47 +30,47 @@ namespace MultiplayerSample
         }
     }
 
-    void UiCanvasDemoPlacardComponent::Deactivate()
+    void DemoPlacardComponent::Deactivate()
     {
         UiCanvasAssetRefNotificationBus::Handler::BusDisconnect(GetEntityId());
     }
 
-    void UiCanvasDemoPlacardComponent::Reflect(AZ::ReflectContext* context)
+    void DemoPlacardComponent::Reflect(AZ::ReflectContext* context)
     {
         if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
-            serializeContext->Class<UiCanvasDemoPlacardComponent, AZ::Component>()
+            serializeContext->Class<DemoPlacardComponent, AZ::Component>()
                 ->Version(1)
-                ->Field("Text", &UiCanvasDemoPlacardComponent::m_placardText)
-                ->Field("TextboxUiId", &UiCanvasDemoPlacardComponent::m_placardTextboxUiElementId)
+                ->Field("Text", &DemoPlacardComponent::m_placardText)
+                ->Field("TextboxUiId", &DemoPlacardComponent::m_placardTextboxUiElementId)
                 ;
 
             if (AZ::EditContext* editContext = serializeContext->GetEditContext())
             {
-                editContext->Class<UiCanvasDemoPlacardComponent>("UiCanvasDemoPlacardComponent", "Helper component for setting up UI canvases placards on meshes in and around the world. This component can dynamically change the placard text so that we can reuse one UiCanvas for multiple placards.")
+                editContext->Class<DemoPlacardComponent>("DemoPlacardComponent", "Helper component for setting up UI canvases placards on meshes in and around the world. This component can dynamically change the placard text so that we can reuse one UiCanvas for multiple placards.")
                     ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
-                        ->Attribute(AZ::Edit::Attributes::Category, "ComponentCategory")
+                        ->Attribute(AZ::Edit::Attributes::Category, "Multiplayer Sample UI")
                         ->Attribute(AZ::Edit::Attributes::Icon, "Icons/Components/Component_Placeholder.svg")
                         ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC_CE("Game"))
```

Comment:
FYI: you can actually make a component a true canvas component with "CanvasUI" then you can set the entities here with drag and drop in LyShine editor and then you can avoid needing to call:

```
AZ::Entity* textBoxEntity;
        UiCanvasBus::EventResult(textBoxEntity, uiCanvasEntityId, &UiCanvasBus::Events::FindElementById, m_placardTextboxUiElementId);
```